SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -565,6 +565,16 @@ int64 USpatialActorChannel::ReplicateActor()
 		}
 	}
 
+	if (Actor->GetIsHiddenDirty())
+	{
+		if (!Actor->GetRootComponent() || !Actor->GetRootComponent()->IsCollisionEnabled())
```

Comment:
Sorry, I am kind of late to the party here, but why is this only updated if there is no root component or it has no collision?

Reply:
yep that was something that I wanted to discuss with Josh as it seems that that's the check that Native is doing to decide if a hidden actor is relevant or not(they also check if it not a root component or it has no collisions), So I am not sure if we want to do the same.